coders/tiff.c

```diff
@@ -2264,7 +2264,7 @@ static Image *ReadTIFFImage(const ImageInfo *image_info,
           }
         image=SyncNextImageInList(image);
         status=SetImageProgress(image,LoadImagesTag,(MagickOffsetType)
-          image->scene-1,image->scene);
```

Comment:
If we still set image->scene to the page number tag, should I also change this back to use the image->scene value?